pkg/cmd/cmd.go

```diff
@@ -86,3 +86,10 @@ func ConfigureNamespaceFlagCompletion(
 			})
 	}
 }
+
+func ConfigureOutputFlagCompletion(cmd *cobra.Command) {
+	cmd.RegisterFlagCompletionFunc("output",
+		func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return []string{"basic", "json"}, cobra.ShellCompDirectiveDefault
```

Comment:
Would `short` work here too?

Reply:
How about including the default `table`?